agents/content/ab_test_orchestrator_agent.py

```
@@ -0,0 +1,796 @@
+                "clicks": m["clicks"]
+            })
+
+        # Simple chi-square approximation
+        # For production, use scipy.stats.chi2_contingency
+        # Here, we use a simplified approach based on CTR variance
+
+        ctrs = [v["ctr"] for v in variant_data]
+        impressions = [v["impressions"] for v in variant_data]
+
+        # Calculate weighted variance (proxy for chi-square)
+        mean_ctr = sum(ctr * imp for ctr, imp in zip(ctrs, impressions)) / sum(impressions)
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)